<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Deduction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="help_btn">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    qproperty-icon: url(src/view/images/help.svg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tab_bar">
      <property name="styleSheet">
       <string notr="true">#scrollAreaContentsBranch {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="single_tab">
       <attribute name="title">
        <string>Single</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">#scrollAreaContentsSingle {
	background-color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaContentsSingle">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>808</width>
             <height>431</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">#scrollAreaWidgetContents {
	background-color: rgb(255, 255, 255);
}</string>
           </property>
           <widget class="QTextEdit" name="deduction_single_left">
            <property name="geometry">
             <rect>
              <x>225</x>
              <y>135</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QTextEdit" name="deduction_single_right">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>135</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="branch_tab">
       <attribute name="title">
        <string>Branch</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaContentsBranch">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>808</width>
             <height>431</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#scrollAreaWidgetContentsBranch {
	background-color: rgb(255, 255, 255);
}</string>
           </property>
           <widget class="QTextEdit" name="deduction_branch_ll">
            <property name="geometry">
             <rect>
              <x>35</x>
              <y>185</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QTextEdit" name="deduction_branch_lr">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>185</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QTextEdit" name="deduction_branch_rl">
            <property name="geometry">
             <rect>
              <x>425</x>
              <y>185</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
           <widget class="QTextEdit" name="deduction_branch_rr">
            <property name="geometry">
             <rect>
              <x>610</x>
              <y>185</y>
              <width>165</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ok_btn">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
